### PR TITLE
Task/remove external deps

### PR DESCRIPTION
**Pull request by dpetran**
This removes the need for external dependencies for our nodejs library.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -2,16 +2,23 @@
...
-    "karma": "^6.3.4",
-    "karma-chrome-launcher": "^3.1.0",
-    "karma-cljs-test": "^0.1.0",
-    "webpack": "^5.44.0",
-    "webpack-cli": "^3.3.12"
+    "karma": "6.3.4",
+    "karma-chrome-launcher": "3.1.0",
+    "karma-cljs-test": "0.1.0",
+    "webpack": "5.51.1",
+    "webpack-cli": "4.8.0"
```
Comment (cap10morgan) [inline package.json:21]: Do we not want the `^` qualifier? I don't have a strong opinion either way, just curious.

File: src/fluree/db/util/xhttp.cljc [hunk 2]
```diff
@@ -65,18 +74,17 @@
...
-(defn throw-if-timeout [response]
-  (if (= TimeoutException (-> response
-                              :error
-                              :error/via
-                              first
-                              :type))
-    (throw (TimeoutException. (-> response :error :error/cause)))
-    response))
-
+#?(:clj
```
Comment (cap10morgan) [inline src/fluree/db/util/xhttp.cljc:77]: 👍🏻

**Review by cap10morgan — APPROVED** [review]
This is great! 🎉

File: package.json (hunk 1 above)
Comment (dpetran) [reply package.json:21]: I removed it while troubleshooting to make sure I was getting the exact version I was asking for, and I generally prefer pinning to a specific version. But I don't think it affects anything concretely here, honestly.

**Pull request merged**